### PR TITLE
TW 436 Tab behavior for web

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

File: src/pages/docs/getting-started/navigating-postman.md [hunk 1]
```diff
@@ -139,12 +139,14 @@ To save request responses in your history, select the more actions icon <img alt
...
+Whether you're working with a collection, an API, or another element type, the center section of Postman is where you do the majority of your work. [Tabs](#tabs) enable you to organize your work, while the [right sidebar](#right-sidebar) gives you access to element-specific tools like documentation. The [environment selector and environment quick look](#environment-selector-and-environment-quick-look) enable you to manage variables.
 
 ### Tabs
 
 Tabs allow you to organize and work between requests.
 
+> This section describes how tabs work in the Postman Desktop app. By default, Postman for Web only shows one collection overview or request in the main work area at a time, rather than opening new in-app tabs as described here. For more information, see [Browser tabs in Postman for Web](#browser-tabs-in-postman-for-web).
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/navigating-postman.md:148]: I'm not sure this is the default; I think it's opt-in and you get a CTA asking you. (But I didn't get the CTA on the beta track, so ¯\_(ツ)_/¯)

File: src/pages/docs/getting-started/navigating-postman.md [hunk 2]
```diff
@@ -191,6 +193,18 @@ If you have a lot of tabs open, they might overflow the area of the tab bar. To
 
 <img alt="Scroll to view tabs" src="https://assets.postman.com/postman-docs/tab-navigation-scroll.jpg" width="800px"/>
 
+#### Browser tabs in Postman for Web
+
+By default, Postman for Web only shows one collection overview or request in the main work area at a time, rather than opening new in-app tabs like the Postman Desktop app does. You can open more Postman elements in new browser tabs, which enables you to work across workspaces or to use different environments in the same workspace.
+
+If you've saved changes to a request, when you click on a new element Postman will open it in the main work area area. If the request has unsaved changes, Postman will prompt you to save or discard your changes.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/getting-started/navigating-postman.md:200]: 🤦

File: src/pages/docs/getting-started/settings.md [hunk 3]
```diff
@@ -75,6 +75,7 @@ When you send a form-data or binary file with a request body, Postman saves a pa
 
 ### User interface
 
+* **Remove tabs** - _Postman for Web only._ Use tabs in your browser to navigate Postman instead of in-app tabs. For more information, see [Browser tabs in Postman for Web](#/docs/getting-started/navigating-postman/#browser-tabs-in-postman-for-web). 
 * **Two-pane view:** By default, Postman displays responses below requests. Turn this on to display the response and request panes side by side.
```
Comment (HeatherStenson) [inline src/pages/docs/getting-started/settings.md:79]: The formatting for other list items is being handled in https://github.com/postmanlabs/postman-docs/pull/4065, but if this merges first I'll update this PR instead

File: src/pages/docs/getting-started/navigating-postman.md [hunk 4]
```diff
@@ -189,6 +191,18 @@ If you have a lot of tabs open, they might overflow the area of the tab bar. To
...
+
+By default, Postman for Web uses in-app tabs like the Postman Desktop app does. You can optionally choose to use browser tabs instead. With this setting turned on, there is only one collection overview or request in the main work area at a time, and you can open more Postman elements in new browser tabs. This enables you to work across workspaces or to use different environments in the same workspace.
+
+If you've saved changes to a request, when you select a new element Postman will open it in the main work area area. If the request has unsaved changes, Postman will prompt you to save or discard your changes.
+
+To open a workspace, environment, or request in a new browser tab, enter **⌘+Shift** or **Ctrl+Shift** and select the element you want to open.
+
+To open an in-app link in a new browser tab, enter **⌘** or **Ctrl** and select the link you want to open.
+
+To turn this behavior on or off in Postman for Web, select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> in the header, then select **Use browser tabs** in the **User interface** section.
```
Comment (HeatherStenson) [inline src/pages/docs/getting-started/navigating-postman.md:204]:
```suggestion
#### Browser tabs in Postman for Web

By default, Postman for Web uses in-app tabs like the Postman Desktop app does. You can optionally choose to use browser tabs instead. With this setting turned on, there is only one collection overview or request in the main work area at a time, and you can open more Postman elements in new browser tabs. This enables you to work across workspaces or to use different environments in the same workspace.

If you've saved changes to a request, when you select a new element Postman will open it in the main work area area. If the request has unsaved changes, Postman will prompt you to save or discard your changes.

To open a workspace, environment, or request in a new browser tab, enter **⌘+Shift** or **Ctrl+Shift** and select the element you want to open.

To open an in-app link in a new browser tab, enter **⌘** or **Ctrl** and select the link you want to open.

To turn this behavior on or off in Postman for Web, select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px"> in the header, then select **Remove tabs** in the **User interface** section.
```

**Pull request merged**